examples/src/contracts/tlw.rs

```diff
@@ -90,7 +90,7 @@ pub enum Error {
 }
 
 /// Deposit event.
-#[derive(Event, PartialEq, Eq, Debug)]
+#[odra::event]
```

Comment:
So maybe for consistency, we should do the same with `OdraType`.
Then instead of `#[derive(OdraType, PartialEq, Debug)]` we would write `#[odra::type]` (or `#[odra::odra_type]`).
We would end up with `#[odra::module]`, `#[odra::type]` and `#[odra::event]`. WDYT?

Reply:
You may look at feature/macro-attr branch

Reply:
This is exactly what I though. Please make it in the next PR.